### PR TITLE
Next.js with-xata example

### PR DESCRIPTION
**Pull request by atilafassina**
- Replace `npm` with `pnpm` to better manage conflicting dependencies across workspaces
- add script to add dummy data to a new DB in Xata

<details>
<summary>screenshots</summary>

<div width="500">

Screen Shot 2022-08-16 at 15 36 06

Screen Shot 2022-08-16 at 15 33 55
</div>

</details>

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+XATA_API_KEY=xau_vS8jN7pzGKpjRvbuumA5GPc0CI8ipBU5b
```
Comment (TejasQ) [inline .env:1]: wtf man

File: .gitignore [hunk 2]
```diff
@@ -42,4 +42,11 @@ Thumbs.db
...
 .turbo
-node_modules
\ No newline at end of file
+node_modules
+
+**/.xatarc
+apps/**/xata.codegen.ts
+.env
+.env.*
+
```
Comment (TejasQ) [inline .gitignore:51]: Why are we ignoring this?

File: apps/nextjs-minimal/.gitignore [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+.DS_Store
+Thumbs.db
+
+.next
+.turbo
+node_modules
+
+**/.xatarc
+.env
+.env.*
```
Comment (TejasQ) [inline apps/nextjs-minimal/.gitignore:49]: Maybe good to not ignore .env.local?

Comment (atilafassina) [reply apps/nextjs-minimal/.gitignore:49]: I think that opens chance to leak the xata token to github as users won't have a local instance to fetch from

Comment (TejasQ) [reply apps/nextjs-minimal/.gitignore:49]: Ok

**Review by TejasQ — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (atilafassina) [reply .gitignore:51]: ignoring the every variation of `.env.` so users need to explicitly decide to commit things if they choose so

maybe I could allow-list `.env.template` and `.env.example` as they are common patterns?

**Review by fabien0102 — APPROVED** [review]
Tested locally, works like a charm 💯

**Pull request merged**